docs/interfaces/_src_methods_balances_transfer_.balancestransferargs.md

```diff
@@ -17,20 +19,20 @@
 
 ## Properties
 
-###  dest
+### dest
 
-• **dest**: *string*
+•  **dest**: string
```

Comment:
On line 22 you remove a space, here you add one. Oh my OCD…

Reply:
This is all auto generated and since the source code file this is generated from was not touched in this PR I suspect the docs generation tool has had some updates that affect spacing.